app/controllers/converts_controller.rb

```diff
@@ -28,6 +28,9 @@ def create
       filename: file_name,
       layout: false
     )
+  rescue ActionController::ParameterMissing
+    flash.now[:alert] = "Missing 'file' or 'format' form option."
```

Comment:
Nicer in terms of language. If I'm a user and I accidentally forget to pick a file to convert, I should be told that I didn't pick a file to convert. Being told that I'm missing a 'file' or 'format' form option is a bit cryptic and wordy.

If everything on the page behaves correctly and users aren't messing with anything, it's impossible to be missing a format, so I only care about the friendliness of the missing file message. Just something simple like "You didn't select a file." or something like that.